.github/workflows/armv7l_wheels.yml

```diff
@@ -0,0 +1,65 @@
+name: armv7l wheels
+
+on:
+  push:
+  pull_request:
+  create:
+  schedule:
+    - cron:  '23 1 * * *'
+
+env:
+  KIVY_SPLIT_EXAMPLES: 1
```

Comment:
`KIVY_SPLIT_EXAMPLES=1` needs to be set in the environment of the docker container because setting it here doesn't get passed into the docker container. See https://github.com/kivy/kivy/pull/6656.

Reply:
See: 6d2ceb51de88ac5830b67cee667f03c28eda7e14